lib/DDG/Spice/Hex.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::Hex;
+
+# ABSTRACT: Returns package information from Hex package manager
+
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d";
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'https://hex.pm/api/packages/$1';
+
+triggers startend => 'hex', 'elixir';
```

Comment:
Elixir seems to have the same problem. As a guitarist Elixir makes me think of the guitar string brand:

![hex_install_httpotion_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/14575785/14ccbdbc-0332-11e6-9426-11510adee176.png)